### PR TITLE
GroupImportFixer - fix removal of import type when mixing multiple types

### LINKED ISSUE #5346: group_import removes import type when mixing multiple types
## Bug report

The `group_import` rule removes the import type when mixing imports of different types.

### Code snippet that reproduces the problem

```php
<?php 
use Foo\Bar;
use function Foo\baz;
```

This will be fixed to
```php
<?php
use Foo\{Bar, baz};
```

while it should be fixed to
```php
<?php
use Foo\{Bar, function baz};
```

### PR DESCRIPTION
**Pull request by Leprechaunz**
Fixes #5346

### CODE REVIEW

File: src/Tokenizer/Analyzer/Analysis/NamespaceUseAnalysis.php [hunk 1]
```diff
@@ -121,6 +121,14 @@ public function getEndIndex()
         return $this->endIndex;
     }
 
+    /**
+     * @return int
+     */
+    public function getType()
```
Comment (kubawerlos) [inline src/Tokenizer/Analyzer/Analysis/NamespaceUseAnalysis.php:127]: Let's have some tests for this one in `NamespaceUseAnalysisTest`.

**Review by kubawerlos — CHANGES_REQUESTED** [review]

**Review by kubawerlos — APPROVED** [review]

**Comment by keradus** [comment]
Hey @Leprechaunz, can you resolve conflict by rebasing the PR, please?

**Comment by Leprechaunz** [comment]
@keradus resolved conflicts ✅

**Comment by keradus** [comment]
Thank you @Leprechaunz.

**Pull request merged**